webpack/components/extensions/HostDetails/Tabs/TracesTab/EnableTracerModal.js

```diff
@@ -43,7 +43,7 @@ const EnableTracerModal = ({ isOpen, setIsOpen, triggerJobStart }) => {
   };
 
   const dropdownItems = dropdownOptions.map(text => (
-    <DropdownItem key={`option_${text}`} onClick={() => setSelectedOption(text)}>{text}</DropdownItem>
+    <DropdownItem key={`option_${text}`} ouiaId={`option_${text}`} onClick={() => setSelectedOption(text)}>{text}</DropdownItem>
```

Comment:
@ianballou Do you see this change in your environment?

Reply:
I do indeed